nvi-rest/src/main/java/no/sikt/nva/nvi/fetch/FetchCandidateResponse.java

```diff
@@ -0,0 +1,38 @@
+package no.sikt.nva.nvi.fetch;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import java.net.URI;
+import java.util.List;
+import no.sikt.nva.nvi.common.model.business.Candidate;
+
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
+@JsonSerialize
+public record FetchCandidateResponse(
+    URI publicationId,
+    List<ApprovalStatus> approvalStatuses,
+    List<Note> notes
+) {
```

Comment:
Is this weird formatting, or are records just like this?

Reply:
Preferred formating?